src/tests/networking_tests.rs

```diff
@@ -1,32 +1,33 @@
 #[cfg(test)]
 pub mod net_tests {
-    use crate::interface;
+    use crate::interface::{self, fd_set_check_fd, fd_set_insert, fd_set_remove};
```

Comment:
why do we have to import these separately?

Reply:
we call functions from the interface in here already without specific imports so don't really understand this change